docs/PYPI.md

```diff
@@ -11,30 +11,38 @@
 ![Lines of code](https://tokei.rs/b1/github/pincer-org/pincer?category=code&path=pincer)
 ![Repo Size](https://img.shields.io/github/repo-size/Pincer-org/Pincer)
 ![GitHub last commit](https://img.shields.io/github/last-commit/Pincer-org/Pincer)
-![GitHub commit activity](https://img.shields.io/github/commit-activity/m/Pincer-org/Pincer)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/m/Pincer-org/Pincer?label=commits)
 ![GitHub](https://img.shields.io/github/license/Pincer-org/Pincer)
 ![Discord](https://img.shields.io/discord/881531065859190804)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-An asynchronous Python API wrapper meant to replace discord.py
+# <img src="../assets/svg/pincer.svg" height="24px" alt="Pincer Logo"> Pincer
+The snappy asynchronous discord api wrapper API wrapper written with aiohttp & websockets.
```

Comment:
Would already remove the `& websockets` part so there's no need to update it later when #329 gets merged, but that's up to you I guess, so use one of these:
```suggestion
The snappy asynchronous Discord API wrapper written with aiohttp & websockets.
```
```suggestion
The snappy asynchronous Discord API wrapper written with aiohttp.
```

Reply:
Good idea, I hate merge conflicts xD